web-marketplace/src/components/organisms/DashboardNavigation/DashboardNavigation.utils.tsx

```diff
@@ -171,17 +171,16 @@ export function getDashboardNavigationSections(
   hasWalletAddress = true,
   hasProjects = false,
   hasOrders = true,
-  walletConnect = false,
   hasCreditBatches = false,
 ): DashboardNavigationSection[] {
   const sections = [];
   if (hasWalletAddress) {
     sections.push(getCreditsSection(_, collapsed));
   }
   if (
-    (walletConnect && hasProjects) ||
-    (!walletConnect && !hasProjects) ||
-    (!walletConnect && hasProjects)
```

Comment:
these old conditions actually always resulted to true it seems
it looks now that the second condition is different: using `loginDisabled` instead of `!walletConnect` whereas everywhere else you use `loginDisabled` in place of `walletConnect` as it should

I think we should rather **hide** the projects section only if loginDisabled && !hasProjects